eq3bt/eq3btsmart.py

```diff
@@ -140,6 +147,20 @@ def handle_notification(self, data):
             else:
                 self._mode = Mode.Auto
 
+            presets = status.presets
+            if presets:
+                self._window_open_temperature = presets.window_open_temp
+                self._window_open_time = presets.window_open_time
+                self._comfort_temperature = presets.comfort_temp
+                self._eco_temperature = presets.eco_temp
+                self._temperature_offset = presets.offset
```

Comment:
Maybe add a debug output for these?

Reply:
Done. I also noticed that I had forgotten to add the regular output to the CLI tool. This is now fixed as well.